docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md

```diff
@@ -0,0 +1,336 @@
+    name: "shield Shield"
+    adb_server_ip: 127.0.0.1
+    adb_server_port: 5037
+    apps:
+      "com.google.android.tvlauncher": "Home"
+      "com.netflix.ninja": "Netflix"
+      "com.google.android.youtube.tv": "Youtube"
+      "com.google.android.tv": "Live Channels"
+      "com.amazon.amazonvideo.livingroom": "Amazon Video"
+      "com.plexapp.android": "Plex"
+      "com.nvidia.bbciplayer": "iPlayer"
+      "com.nvidia.bbciplayer.launch": "BBC News"
+      "com.silicondust.view": "HD HomeRun"
+```
+
+### Create a script 
+
+To launch Android TV apps from home Assistant you need to create a script. I have created some examples below. 
+
+
+```YAML
+script:
+
+## Leanback Launcher
+  adb_open_shield_leanback:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: HOME
+
+## Plex
+  adb_open_shield_plex:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "monkey -p com.plexapp.android -c android.intent.category.LAUNCHER 1"
+
+## Netflix
+  adb_open_shield_netflix:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "am start -a android.intent.action.VIEW -d -n com.netflix.ninja/.MainActivity"
+
+## Youtube
+  adb_open_shield_youtube:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "am start -a android.intent.action.VIEW -d -n com.google.android.youtube.tv/com.google.android.apps.youtube.tv.activity.ShellActivity"
+
+## Amazon Video
+  adb_open_shield_amazon:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "am start -a android.intent.action.VIEW -d -n com.amazon.amazonvideo.livingroom/com.amazon.ignition.IgnitionActivity"
+
+## iPlayer
+  adb_open_shield_iplayer:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command:  "am start -a android.intent.action.VIEW -d -n com.nvidia.bbciplayer/.MainPlayerActivity"
+
+## BBC News
+  adb_open_shield_bbc_news:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "am start -a android.intent.action.VIEW -d -n com.nvidia.bbciplayer.launch/com.nvidia.bbciplayer.LaunchNewsActivity"
+
+## HD Homerun
+  adb_open_shield_hd_hr:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "am start -a android.intent.action.VIEW -d -n com.silicondust.view/.App"
+```
+
+Further working examples can be found on my [GitHub](https://github.com/noodlemctwoodle/homeassistant)
```

Comment:
Not sure I like the use of the word "my" here, feels a bit odd when there's no noted author.

```suggestion
[This repo](https://github.com/noodlemctwoodle/homeassistant) contains further working examples.
```

Reply:
changed sentance to `Further working examples can be found on this `